scripts/presubmit.sh

```diff
@@ -97,27 +97,31 @@ main() {
     go build ${goflags} ./...
 
     echo 'running go test'
+    # Install test deps so that individual test runs below can reuse them.
+    echo 'getting test deps'
+    go get -t
+    echo 'installed test deps'
+    go test -i ./...
 
+    if [[ ${coverage} -eq 1 ]]; then
     # Individual package profiles are written to "$profile.out" files under
     # /tmp/trillian_profile.
     # An aggregate profile is created at /tmp/coverage.txt.
     mkdir -p /tmp/trillian_profile
     rm -f /tmp/trillian_profile/*
 
     for d in $(go list ./...); do
-      # Create a different -coverprofile for each test (if enabled)
+      # Create a different -coverprofile for each test
       local coverflags=
```

Comment:
Can delete this line, and just add `local` to line 124.

Reply:
done